### PR TITLE
Bump react-scripts to 5.0.0

### PR DESCRIPTION
**Pull request by pankona**
Closes: https://github.com/mobu-of-the-world/mobu/pull/277

## Updates

- Fix eslint error caused by bumping react-scripts to 5.0.0

## Remarks

- Single parameter generics cannot declare in jsx (tsx)
  - A comma after the type parameter resolves the issue. Refs: https://github.com/Microsoft/TypeScript/issues/15713#issuecomment-499474386

### CODE REVIEW

**Comment by pankona** [comment]
@mobu-of-the-world/maintainers 
Please review and merge if there's no concern! 😋

File: src/pages/AppContainer.tsx [hunk 1]
```diff
@@ -206,7 +206,7 @@ const numberToTimeString = (count: number): string => {
   return elapsedTime.toISOString().substr(11, 8);
 };
 
-const shuffleArray = <T extends unknown>(array: T[]): T[] => {
+const shuffleArray = <T,>(array: T[]): T[] => {
```
Comment (kachick) [inline src/pages/AppContainer.tsx:209]: 😮

**Review by kachick — APPROVED** [review]
Thank you! 👍

**Comment by kachick** [comment]
yarn.lock in conflicting...

@pankona Could you update?

**Comment by pankona** [comment]
@kachick 

Updated! please review & merge again! 😋

**Comment by pankona** [comment]
@kachick 

Oh, sorry, you already give approve!
I'll go for merge. Thanks! 😋

**Pull request merged**